app/src/main/java/com/spendster/presentation/login/ServerLoginModel.java

```diff
@@ -0,0 +1,14 @@
+package com.spendster.presentation.login;
+
+import com.spendster.data.entity.User;
+import com.spendster.presentation.APIClient;
+
+import io.reactivex.Single;
+
+public class ServerLoginModel implements LoginModel {
```

Comment:
@SimonMaha strange naming. Probably better one would be Simple... 
SimpleLoginModel.